source/connect/favorite-connections/import-export-ui/import.txt

```diff
@@ -0,0 +1,53 @@
+.. _compass-import-connections-ui:
+
+==============================================
+Import Favorite Connections on |compass-short|
+==============================================
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can import favorite connections on |compass|. This lets you use saved 
+connections from other workspaces or connections used by other team members.
+
+Prerequisite
+~~~~~~~~~~~~
+
+Before you import favorite connections, you must :ref:`export the
+connections <compass-export-connections-ui>`.
+
+Procedure
+~~~~~~~~~
+
+To import favorite connections from a JSON file into |compass-short|:
+
+.. procedure::
+    :style: normal
+    
+    .. step:: Hover over :guilabel:`Saved Connections` in the left-side 
+        navigation and click the :guilabel:`Ellipses (...)` button.
+
+    .. step:: Select :guilabel:`Import saved connections` to open the import 
+        dialog box.
+
+    .. step:: Under :guilabel:`Source File`, click :guilabel:`Select a file...` 
+        to select the exported JSON file you want to import.
+
+    .. step:: If your file is encrypted with a password, enter the passphrase 
+        under :guilabel:`Decryption Password`.
+    
+    .. step:: Select the connection names you want to 
+        import. 
+        
+        If the connection is already saved, |compass-short| overwrites the 
+        existing connection with the imported connection.
+
+    .. step:: Click 
+        :guilabel:`Import`
```

Comment:
Note: I had to add this somewhat awkward newline to preserve consistent styling across each step. Otherwise, this step would for some reason become its own header, that appears in the in-page TOC.

Reply:
I'll bring it up with them! I just didn't before bc I couldn't find another ticket/PR where the inconsistent styling was happening so not sure if it's a one-off issue.

Reply:
Just a follow-up to this, looks like it's been resolved based on the staging! Might've been a temporary bug that came out of some internal changes they could've pushed out.